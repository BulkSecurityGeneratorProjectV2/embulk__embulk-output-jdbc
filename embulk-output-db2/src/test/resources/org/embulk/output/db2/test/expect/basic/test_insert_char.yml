table: TEST_CHAR
mode: insert
