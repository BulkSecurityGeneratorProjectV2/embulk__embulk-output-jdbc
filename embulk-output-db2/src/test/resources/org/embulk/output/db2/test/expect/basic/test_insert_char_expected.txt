ID CHAR_ITEM  VARCHAR_ITEM  CLOB_ITEM                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               GRAPHIC_ITEM  VARGRAPHIC_ITEM  NCHAR_ITEM          NVARCHAR_ITEM
1001 aa         AA            aaaaaaaaaaaa                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            ああ          いいいい          ａａ                ＡＡ
1002 XXXX       XXXXXXXX      XXXXXXXXXXXXXXXX                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        XXXX          XXXXXXXX         XXXX                XXXXXXXX
1003
9999
------------- ---------- ------------- --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- ------------- ---------------- ------------------- -------------------------------------
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
NCLOB_ITEM
XXXXXXXXXXXXXXXX
ａａａａａａａａ
